crm_referred/readme/CONTEXT.md

```diff
@@ -0,0 +1,3 @@
+This module was developed because field referred by from Odoo Core is a char field and salesperson can write anything, then leads/opportunities cannot be categorised.
```

Comment:
```suggestion
This module was developed because field *Referred by* from Odoo Core is a char field and salesperson can write anything, then leads/opportunities cannot be categorised.
```